docs/source/configuration/telemetry/exporters/tracing/overview.mdx

```diff
@@ -144,9 +144,28 @@ telemetry:
    
          # If you have your own way to generate a trace id and you want to pass it via a custom request header
          request:
+           # The name of the header to read the trace id from
            header_name: my-trace-id
+           # The format of the trace when propagating to subgraphs.
+           format: uuid
 ```
 
+#### `request` configuration reference
+
+| Option        | Values                                                        | Default                           | Description                         |
+|---------------|---------------------------------------------------------------|-----------------------------------|-------------------------------------|
+| `header_name` |                                                               | Enable or disable stdout logging. |                                     |
```

Comment:
Is this a boolean? Should its value be `boolean` or `true|false`?

Reply:
Oops the description is wrong.